lab/server/package.json

```diff
@@ -12,8 +12,9 @@
   "license": "MIT",
   "dependencies": {
     "cors": "^2.8.4",
-    "express": "^4.16.2",
-    "pg": "^7.3.0",
-    "superagent": "^3.8.0"
+    "express": "^4.16.3",
+    "fs": "0.0.1-security",
```

Comment:
You shouldn't need to install fs explicitly... it should be built into Node. Hopefully you're just running `npm install` when you get into this directory, and not explicitly running `npm install express`, `npm install pg`, etc?